packages/bpk-tokens/src/android/base/typography.json

```diff
@@ -179,7 +179,7 @@
       "category": "font-sizes"
     },
     "TEXT_XL_FONT_WEIGHT": {
-      "value": "{!FONT_WEIGHT_REGULAR}",
+      "value": "{!FONT_WEIGHT_MEDIUM}",
```

Comment:
The PR title talks about `lg` and `base`, but you seem to have changed `xl` and `base`??